app/templates/admin/createSingleEvent.html

```diff
@@ -149,8 +154,13 @@ <h1 id="title">{{page_title}}</h1>
               <label class="form-label me-2" for="pickEndTime"><strong>End Time</strong></label>
               {% if isEditable %}
               <div class="input-group">
-                <input autocomplete="off" type="time" class="form-control timepicker readonly" value="{{eventData.timeEnd}}" name="timeEnd" aria-describedby="timeIcon2" placeholder="Pick an end time" onchange="update(this)" id=endTime required />
-                <span class="input-group-text timeIcons" id="timeIcon2" hidden><i class="bi bi-clock"></i></span>
+                {% if eventData.timeEnd %}
+                  {%set end_time = eventData.timeEnd%}
+                {%else%}
+                  {%set end_time = "23:59" %}
```

Comment:
I think this should also be an even hour, so our time picker doesn't default to 59 minutes